robocup_ws/src/GameEngine/src/ball_model.py

```diff
@@ -305,7 +305,7 @@ def _kick_collision(self, collision_object, kick_vel: float = 0.4, **kwargs):
         player_pos_x, player_pos_y = collision_object.get_position_components_wcs()
         dx, dy = self._x_pos - player_pos_x, self._y_pos - player_pos_y
         diff = np.hypot(dx, dy)
-        if diff < 0.17:
+        if diff < 0.2:
```

Comment:
if you really need to modify it add constant ```ball_in_contact_range``` and refer to it here ```self. ball_in_contact_range```

Btw: this should be 0.15 - so I would prefer to keep it 0.17 or even change it to 0.15

Reply:
Maybe now we can use 0.2 first, at least, we know which part has conflicts

Reply:
have you tried lowering this value: https://github.com/szgula/UofG_Robotics_TDP/pull/75#discussion_r593799126 ?